docs/config/README.md

```diff
@@ -437,6 +437,31 @@ The module will be shown if any of the following conditions are met:
 symbol = "⚙️ "
 ```
 
+## Time
+
+The `time` module shows the current **local** time.
+
+::: tip
```

Comment:
I would personally suggest putting the details about formatting in the description above. I think the tip box would be better suited to remind the user that this module is "disabled by default".

Reply:
Done! (https://github.com/starship/starship/pull/138/commits/7727b90ddad41784aa6ea39421bc28a578bd3bdc)